application-forum-ui/src/main/resources/ForumCode/ConfigurationAdmin.xml

```diff
@@ -40,20 +40,22 @@
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
   <content>{{include reference="Licenses.Code.VelocityMacros"/}}
-
+{{include reference='ForumCode.Macros'/}}
 {{velocity}}
 #set ($mainReference = $services.model.createDocumentReference('', 'ForumCode', 'ConfigurationClass'))
 #if (!$services.licensing.licensor.hasLicensureForEntity($mainReference))
 
   {{error}}#getMissingLicenseMessage('forum.extension.name'){{/error}}
 
 #else
-  {{include reference="ForumCode.FlagsHome"/}}
-
+  {{include reference='ForumCode.FlagsHome'/}}
   == $services.localization.render('forum.administration.configuration.title') ==
+  #checkIfHostNameIsValid($doc.getExternalURL(), $result)
+  #if ($result == 'false')
+    {{warning}}$services.localization.render('forum.administration.warningMessage'){{/warning}}
```

Comment:
``warningMessage`` is too generic. I'd rather use ``forum.administration.xxxWarning``, e.g. ``forum.administration.cannotShareURLWarning`` (a suggestion).